### ISSUE TITLE
Anaconda: PyWin32>=227: Entry Point Not Found  PyWinObject_FROMULARGE_INTEGER

### ISSUE THREAD

**Issue by AndrewTsao** [mhammond/pywin32#1583]
Note that issues in this repository are only for bugs or feature requests in the pywin32.

**If you need support or help using this package, please follow [these instructions](https://github.com/mhammond/pywin32/blob/master/README.md#support)** - support or help requests will be closed without comment.

For all bugs, please provide the following information.

* Expected behavior and actual behavior.

```
> pip show
```

alert a message box show

```
Entry Point Not Found  PyWinObject_FROMULARGE_INTEGER
```

* Steps to reproduce the problem.

* Version of Python and pywin32

```bat
python -mvenv .venv
.venv\Script\activate
pip install pywin32==227
pip show
```

```bat
(.venv) C:\Users\xxxx\test>python
Python 3.6.3 |Anaconda, Inc.| (default, Oct 15 2017, 03:27:45) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
```
os: windows server 2008 r2.

and `pywin32<=226` is good.

**Comment by mhammond** [comment]
This is because anaconda includes an older version of pywin32 and stores the DLLs in non-standard places. There's literally nothing I can do to fix this :( See also #1409 (searching for "anaconda" in the open issues will also show some others)

**Comment by AndrewTsao** [comment]
Thank you for you reply.
